src/pages/Manager.vue

```diff
@@ -490,9 +490,12 @@ import ModalCard from '../components/ModalCard.vue';
                 case PackageLoader.MELON_LOADER:
                     logOutputPath = path.join(this.profile.getProfilePath(), "MelonLoader", "Latest.log");
                     break;
-				case PackageLoader.RETURN_OF_MODDING:
```

Comment:
Nitpick: this file unfortunately mixes tabs and spaces and no one's dared to refactor it. Anyway the indentation looks a bit wonky now.